### DISCUSSION TITLE
Natural Weapons

### DISCUSSION THREAD

**Discussion by LouderthanThunder** [Lambosaurus/hives-and-torches#46] [Feats]
Gain two claw and a bite attack 

2d4+d6? 

Cannot use any weapons while using natural attacks

Cannot be disarmed, cannot fumble Etc

Prototype Druid abilities

**Comment by Lambosaurus** [comment]
The balance seems fine, but we don't have a class for it. Unless druid can be a zealot progression? Alternately it could be a racial.
